components/esp_lvgl_port/CMakeLists.txt

```diff
@@ -77,12 +77,12 @@ if("usb_host_hid" IN_LIST build_components)
 endif()
 
 # Include SIMD assembly source code for rendering, only for (9.1.0 <= LVG_version < 9.2.0) and only for esp32 and esp32s3
-if((lvgl_ver VERSION_GREATER_EQUAL "9.1.0") AND (lvgl_ver VERSION_LESS "9.2.0"))
-    if(CONFIG_IDF_TARGET_ESP32 OR CONFIG_IDF_TARGET_ESP32S3)
+if((lvgl_ver VERSION_GREATER_EQUAL "9.1.0") AND (lvgl_ver VERSION_LESS_EQUAL "9.2.0"))
```

Comment:
Does it really work with 9.2.0? we had some compatibility problems with this version. If yes, why can't support 9.2.2 too?